anago

```diff
@@ -857,10 +857,11 @@ EOF
 ###############################################################################
 # generate the announcement text to be mailed and published
 release_announcement_text () {
+  local GOLANG_VERSION=$(go version)
   cat <<EOF
 Kubernetes Community,
 <P>
-Kubernetes $RELEASE_VERSION_PRIME has been built and pushed.
+Kubernetes $RELEASE_VERSION_PRIME has been built and pushed using Golang version $GOLANG_VERSION.
```

Comment:
`go version` already outputs something like `go version go1.14.6 linux/amd64`. We probably would like to strip away the build arch (which might be misleading) and:
```suggestion
Kubernetes $RELEASE_VERSION_PRIME has been built and pushed using $GOLANG_VERSION.
```

Reply:
thanks for the review. Done, PTAL

Reply:
Can we also strip the `linux/amd64` (or equivalent arch) suffix?

Reply:
done! good call!